imagersite/imager_api/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.conf.urls import url, include
+from rest_framework import routers
+from . import views
+
+router = routers.DefaultRouter()
+router.register(r'api/v1', views.PhotoViewSet)
```

Comment:
This is one resource that you might register with this api, but what about others?  You might want to name this resource better.